pkg/util/screenshot/screenshot.go

```diff
@@ -0,0 +1,32 @@
+package screenshot
+
+import (
+	"fmt"
+	"image"
+	"image/png"
+	"os"
+
+	"github.com/kbinani/screenshot"
+)
+
+func CaptureScreen(outputPath, outputFilename string) error {
+	return CaptureBounds(screenshot.GetDisplayBounds(0), outputPath, outputFilename)
+}
+
+func CaptureBounds(rect image.Rectangle, outputPath, outputFilename string) error {
+	bounds := screenshot.GetDisplayBounds(0)
+	img, err := screenshot.CaptureRect(bounds)
+	if err != nil {
+		return fmt.Errorf("error capturing the screen: %v", err)
+	}
+	file, err := os.Create("test.png")
```

Comment:
outputfileName should be here?

Reply:
yeah yeah need to add the parameters to customize the behave. This was a PoC with fixed values  :)

Reply:
thx for checking though!

Reply:
fixed